src/mad_spark_alt/core/robust_json_handler.py

```
@@ -0,0 +1,197 @@
+    
+    # Fix single quotes
+    text = text.replace("'", '"')
+    
+    # Fix unquoted keys
+    text = re.sub(r'(\w+):', r'"\1":', text)
+    
+    # Remove comments
+    text = re.sub(r'//.*$', '', text, flags=re.MULTILINE)
+    text = re.sub(r'/\*.*?\*/', '', text, flags=re.DOTALL)
+    
+    return text
```

Ruff (0.11.9)

Unnecessary assignment to `text` before `return` statement

Remove unnecessary assignment

(RET504)